pkg/util/kernel/fs.go

```diff
@@ -98,3 +98,60 @@ func IsDebugFSMounted() (bool, error) {
 
 	return true, nil
 }
+
+// isTraceFSMounted would test the existence of file /sys/kernel/tracing/kprobe_events to determine if tracefs is mounted (and debugfs is not)
+// returns a boolean and a possible error message
+func isTraceFSMounted() (bool, error) {
+	_, err := os.Stat("/sys/kernel/tracing/kprobe_events")
+	if err != nil {
+		if os.IsPermission(err) {
+			return false, fmt.Errorf("eBPF not supported, does not have permission to access tracefs")
```

Comment:
It seems we always silent the error (`isMounted, _ := isTraceFSMounted()`).
Should we simply log the errors here instead of returning them ?

Reply:
Thank you, suggestion applied